packages/node_modules/nav-frontend-toggle/src/toggle-gruppe-pure.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import * as cn from 'classnames';
+import { guid, omit } from 'nav-frontend-js-utils';
+import ToggleKnappPure, { ToggleKnappPureProps } from './toggle-knapp-pure';
+
+export interface ToggleGruppePureProps {
+    /**
+     * Array av toggle knapper, se `toggle-knapp-pure.tsx`
+     */
+    toggles?: ToggleKnappPureProps[];
+    /**
+     * Egendefinert klassenavn.
+     */
+    className?: string;
+    /**
+     * Hvis 'true' reduseres høyre/venstre-padding på knappene betraktelig.
+     */
+    kompakt?: boolean;
+    /**
+     * Custom onChange handler
+     */
+    onChange?: (event: React.SyntheticEvent<EventTarget>, index: number) => void;
+}
+
+class ToggleGruppePure extends React.PureComponent<ToggleGruppePureProps> {
+
+    static Toggle = ToggleKnappPure;
+
+    handleClick = (e, index, customOnClick) => {
+        if (typeof customOnClick === 'function') customOnClick(e);
+        if (typeof this.props.onChange === 'function') this.props.onChange(e, index);
+    }
+
+    renderToggles() {
+        if (this.props.children) {
+            return React.Children.map(
+                this.props.children,
+                (child: React.ReactElement<ToggleKnappPureProps>, i: number) => {
+                    if (React.isValidElement(child)) {
+                        return React.cloneElement(child, {
+                            kompakt: this.props.kompakt,
+                            onClick: (e) => this.handleClick(e, i, child.props.onClick)
+                        });
+                    }
+                    return child;
+                }
+            );
+        }
+
+        if (this.props.toggles) {
```

Comment:
Trenger vi egentlig å støtte begge varianter (children og toggles gjennom props)? Det gir bare mer vedlikehold og kompleksitet i nav-frontend sånn jeg ser det, det er løst på én måte andre steder og tenker at det bør holde her også. Stemmer for å begrense det til `toggles`, så slipper vi unna kompleksiteten over ☝️

Reply:
Det mønsteret henger vel igjen fra hvordan Tabs ble implementert. Ikke noe stor fan av å måtte støtte begge variantene jeg heller, så vi kan godt begrense til bare `toggles` for min del 👍